tools/import-image-tests

```diff
@@ -6,26 +6,20 @@ Import Composer image tests
 import argparse
 import json
 import os
+import pathlib
+from typing import Iterator, Tuple, Dict, List, Any, Union
 
-from typing import Iterator, Tuple
+import yaml
 
 
 RESET = "\033[0m"
 GREEN = "\033[32m"
 BOLD = "\033[1m"
 RED = "\033[31m"
+YELLOW = "\033[33m"
 
 
-def parse_name(name: str):
-    comps = name.split("-")
-    distro = comps[0]
-    arch = comps[1]
-    name = comps[2]
-    extra = comps[3] if len(comps) > 3 else None
-    return distro, arch, name, extra
-
-
-def list_tests(source) -> Iterator[Tuple[str, str]]:
+def list_tests(source: Union[str, pathlib.Path]) -> Iterator[Tuple[str, str]]:
```

Comment:
Heads up, using `Iterator` is `Deprecated since version 3.9`. It is a bit annoying. I think `typing.Generator` is the safest replacement.